scripts/configure_dev.sh

```diff
@@ -69,6 +69,13 @@ function install_windows_shellcheck() {
     return 0
 }
 
+function install_python_packages() {
+    pip3 install \
+        argparse \
```

Comment:
argparse is standard library and shouldn't need pip install. swapping in standard library json then removes this entire need for pip doing anything to Python

Reply:
Thanks! I removed this and switched to the json lib